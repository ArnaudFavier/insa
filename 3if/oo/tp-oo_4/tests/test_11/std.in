S segment -3 1 5 1
R rectangle -8 4 4 -4
PC poly 1 1 -1 2 -3 0 -1 -3 1 0
OR union segment rectangle poly
OI inter segment rectangle poly
OR super union inter
UNDO
LIST
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
LIST
REDO
LIST
REDO
REDO
REDO
REDO
REDO
REDO
LIST
UNDO
MOVE segment 1 1
REDO
LIST
UNDO
LIST
REDO
LIST
SAVE canvas.txt
CLEAR
LIST
UNDO
LIST
REDO
LOAD canvas.txt
LIST
UNDO
LIST
REDO
LIST
EXIT
