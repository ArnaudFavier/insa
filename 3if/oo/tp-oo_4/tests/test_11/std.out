OK
OK
OK
OK
OK
OK
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
OK
OK
OK
OK
ERR
OK
segment S [(-3;1) (5;1)]
OK
OK
OK
OK
OK
ERR
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
super U [U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]] I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
OK
ERR
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-2;2) (6;2)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-2;2) (6;2)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
OK
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-2;2) (6;2)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-2;2) (6;2)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-2;2) (6;2)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
